lib/board.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :board
```

Comment:
`attr_accessor` is not allowed. Please, indicate whether is **attr_reader or attr_writer**

Reply:
attr_accessor Is allowed. Sorry.